README.md

```diff
@@ -225,11 +225,14 @@ A number of options are available to allow you to customise the SDK:
 
 - **`steps {List} optional`**
 
-  [link-to-steps-to-components-map]:src/components/App/StepComponentMap.js
+  List of the different steps in the flow. Each step is defined by a named string. The available steps are:
```

Comment:
"Steps in the flow" seems like internal jargon, maybe we could add that each step will display a specific component or add a section with screenshots for each step. What do you guys think?